tasks/phylogenetic_inference/augur/task_augur_mutation_context.wdl

```diff
@@ -0,0 +1,37 @@
+version 1.0
+
+task mutation_context {
+  input {
+    File refined_tree
+    File ancestral_nt_muts_json
+    String build_name
+    
+    Int cpu = 4
+    Int memory = 64
```

Comment:
4 cpus & 64 GB memory feels like too much. Was this required to get this running successfully? My guess is that this task/python script only needs 1 cpu and 4 GB memory at most

Reply:
could also reduce the default disk size to 50GB. I doubt that the input files would ever be larger than 1 GB disk space